ios/Classes/PurchasesFlutterPlugin.m

```diff
@@ -479,10 +493,25 @@ - (void)closeWithResult:(FlutterResult)result {
 #pragma mark Delegate Methods
 
 - (void)purchases:(RCPurchases *)purchases didReceiveUpdatedPurchaserInfo:(RCPurchaserInfo *)purchaserInfo {
-    [self.channel invokeMethod:RNPurchasesPurchaserInfoUpdatedEvent
+    [self.channel invokeMethod:PurchasesPurchaserInfoUpdatedEvent
                      arguments:purchaserInfo.dictionary];
 }
 
+- (void)purchases:(RCPurchases *)purchases
+shouldPurchasePromoProduct:(SKProduct *)product
+   defermentBlock:(RCDeferredPromotionalPurchaseBlock)makeDeferredPurchase {
```

Comment:
indentation is off in these three lines

Reply:
yeah i didn't know how this one was to be handled since the second parameter is longer. my xcode doesn't seem to have any formatting set up, is there a file we use or something?